apps/common/behaviours/custom-validation.js

```diff
@@ -3,6 +3,11 @@ const validators = require('hof/controller/validation/validators');
 module.exports = superclass => class extends superclass {
   validateField(key, req) {
     const validationErrorFunc = (type, args) => new this.ValidationError(key, { type: type, arguments: [args] });
+    function isValidphoneNumber(phoneNumber) {
+      const phoneNumberWithoutSpace = phoneNumber.replace(/\s+/g, '').trim();
+      const isValid = validators.regex(phoneNumberWithoutSpace, /^\(?\+?[\d()-]{8,16}$/);
+      return isValid;
```

Comment:
You can just

```suggestion
      return validators.regex(phoneNumberWithoutSpace, /^\(?\+?[\d()-]{8,16}$/);
```

if you like